specs/plugin-extensibility.md

```diff
@@ -312,7 +312,14 @@ All response attributes are required.
 }
 ```
 
-*signingAlgorithm* : One of following [supported signing algorithms](../signature-specification.md#algorithm-selection), Notation uses this validate the signature, and to set the appropriate attribute in signature envelope (e.g. JWS `alg`). `RSASSA_PSS_SHA_256`, `RSASSA_PSS_SHA_384`, `RSASSA_PSS_SHA_512`,  `ECDSA_SHA_256`, `ECDSA_SHA_384`, `ECDSA_SHA_512`.
+*signingAlgorithm* : One of following [supported signing algorithms](../signature-specification.md#algorithm-selection), Notation uses this validate the signature, and to set the appropriate attribute in signature envelope (e.g. JWS `alg`). Supported values are
+
+* `PS256`: RSASSA-PSS with SHA-256
+* `PS384`: RSASSA-PSS with SHA-384
+* `PS512`: RSASSA-PSS with SHA-512
+* `ES256`: ECDSA on secp256r1 with SHA-256
+* `ES384`: ECDSA on secp384r1 with SHA-384
+* `ES512`: ECDSA on secp521r1 with SHA-512
```

Comment:
I'd prefer the verbose signing algorithm names like `RSASSA-PSS-SHA-256`, `ECDSA-SHA-256` or similar. It also is more in line with alg names like `RSASSA-PSS` in RFC and other docs e.g. [1](https://www.rfc-editor.org/rfc/rfc8017.html#section-8.1), [2](https://datatracker.ietf.org/doc/html/rfc4055#section-3).

Reply:
In [2](https://datatracker.ietf.org/doc/html/rfc4055#section-3) (i.e. RFC 4055 sec 3), the names are
- `rSASSA-PSS-SHA256`
- `rSASSA-PSS-SHA384`
- `rSASSA-PSS-SHA512`

Those names do not look consistent with `SHA-256`, `SHA-384`, and `SHA-512`.

Reply:
Some interesting names...

From [AWS KMS](https://docs.aws.amazon.com/kms/latest/APIReference/API_Sign.html)
- `RSASSA_PSS_SHA_256`
- `RSASSA_PSS_SHA_384`
- `RSASSA_PSS_SHA_512`
- `RSASSA_PKCS1_V1_5_SHA_256`
- `RSASSA_PKCS1_V1_5_SHA_384`
- `RSASSA_PKCS1_V1_5_SHA_512`
- `ECDSA_SHA_256`
- `ECDSA_SHA_384`
- `ECDSA_SHA_512`
- `SM2DSA`

From [Azure Key Vault](https://docs.microsoft.com/en-us/rest/api/keyvault/keys/sign/sign?tabs=HTTP#jsonwebkeysignaturealgorithm)
- `ES256`
- `ES256K`
- `ES384`
- `ES512`
- `PS256`
- `PS384`
- `PS512`
- `RS256`
- `RS384`
- `RS512`
- `RSNULL`